test/tools/ossfuzz/strictasm_diff_ossfuzz.cpp

```diff
@@ -50,6 +50,9 @@ extern "C" int LLVMFuzzerTestOneInput(uint8_t const* _data, size_t _size)
 
 	string input(reinterpret_cast<char const*>(_data), _size);
 
+	if (std::any_of(input.begin(), input.end(), [](char c) { return static_cast<unsigned char>(c) > 127; }))
```

Comment:
Why not restrict to the specific character set? Also, `&& isprint(c)` could be helpful.

Reply:
I wanted to include new line and tab characters, but you are right that the range here is somewhat excessive.

We could use `isprint(c) || '\n' || '\t'` I suppose.

`isprint` gives us the following
  - digits (0123456789)
  - uppercase letters (ABCDEFGHIJKLMNOPQRSTUVWXYZ)
  - lowercase letters (abcdefghijklmnopqrstuvwxyz)
  - punctuation characters (!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~)
  - space ( )
(a total of 95 characters)

New line (`\n`) and horizontal tab ('\t') would make a total of 97 characters in such a reduced character set. What do you think of this?

Reply:
The documentation says that `isprint` might return true for values higher that 127, so probably better to also have the check there. Otherwise looks good!

Reply:
ack. Also, squashed and rebased. 

`isprint(c) || '\n' || '\t'` boosts the parsing success ratio from 41% to 55% :)